src/menu/MenuSongSelection.as

```diff
@@ -803,7 +807,7 @@ package menu
                 }
                 else
                 {
-                    if (options.infoTab == TAB_PLAYLIST)
+                    if (isLocked == false && options.infoTab == TAB_PLAYLIST)
```

Comment:
Couldn't this just use `tarSongItem.isLocked` instead of adding a boolean for the e_onClick?
I might end up removing the e_onClick function all together when I add in-game song purchases which is only used currently to pop the store url open.

Reply:
You're correct, using the `isLocked` property works here.